app/build.gradle

```diff
@@ -112,9 +112,9 @@ dependencies {
     // architecture components
     implementation 'androidx.lifecycle:lifecycle-extensions:2.0.0'
     //room
-    implementation 'androidx.room:room-runtime:2.0.0'
-    kapt 'androidx.room:room-compiler:2.0.0'
-    implementation 'android.arch.persistence.room:rxjava2:1.1.1'
+    implementation 'androidx.room:room-runtime:2.1.0-alpha04'
```

Comment:
I think that you can return `Single` from the inception:
https://medium.com/androiddevelopers/room-rxjava-acb0cd4f3757
(post from June 207)
So no need for alpha, really

Reply:
nope, in the article you linked: 

> Later edit 2: starting with Room 2.1.0-alpha01, DAO methods annotated with @Insert, @Delete or @Update support Rx return types Completable, Single<T> and Maybe<T>.

Reply:
Ah, sorry, I misunderstood, this are mutation functions! They've been always blocking and now you can subscribe on another thread, that's really great.